hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockInfo.java

```diff
@@ -146,7 +146,7 @@ BlockInfo getNext(int index) {
     BlockInfo info = (BlockInfo)triplets[index*3+2];
     assert info == null || info.getClass().getName().startsWith(
         BlockInfo.class.getName()) :
-        "BlockInfo is expected at " + index*3;
+        "BlockInfo is expected at " + (index*3+2);
```

Comment:
Same as above, Can you pad some space between the values

Reply:
OK, I fixed this.